docs(storybook): add react starter guide (#663)
inovex/elements: packages/storybook/src/stories/docs/framework-integration/react-instructions.stories.mdx

As the integration is now complete, you can use the components in your project in the same way you would use any other React component.

To help you to get started, we created a [simple todo app](https://codesandbox.io/s/github/inovex/elements-example-react) for you to play around with.

# Create your first Project

**Lets get you up to speed quickly with the usage of the Elements to create a simple login page.**

First we create a new react app for our login page.

```cmd
npx create-react-app <your app name>
cd <your app name>
npm start
```

Once our react app is ready, delete the boilerplate code and restructure the function until our App.js looks something like this.

> > Hint: For the Styling and Structure of your App we recommend following along with the provided codesandbox.

Reviewer: I would move it below the following code example

```jsx
function App() {
  return <div></div>;
}

export default App;
```

To demonstrate how fast you can create a stylish login form, lets start with our sign in consisting of two input fields for the credentials and one button for the submit.

```jsx
<div className="body">

>   <h1>Login to your Account</h1>

Reviewer: I would use tabs for indentations

  <InoInput label="email" outline></InoInput>
  <InoInput label="password" outline></InoInput>
  <InoButton type="submit">Sign In</InoButton>
</div>
```

## Properties

> To configure the components of the inovex-elements we use **properties**.

Reviewer: ```suggestion
To configure either the functionality or the appearance of the inovex-elements, we use **properties**.
```

> In this case we give our `<InoInput>` the `outline` property and its corresponding label. More properties of the `<InoInput>` can be found in our **[docs](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--playground)**.

Reviewer: ```suggestion
In this case, we give our `<InoInput>` the `outline` property to change its appearance and use the `label`  property to give it its corresponding label. More properties of the `<InoInput>` can be found in our **[docs](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--playground)**.
```

`<InoInput label="email" outline={true}>`.

> > Hint: We can customize almost all Elements components. Add icons to the `<InoInput>` element by adding a [slot and an icon](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--icons) of the `<InoIcon>` component. Play around with each components playground to see all possibilites.

Reviewer: ```suggestion
> Hint: We can customize almost all our components. e.g. add some icons to the `<InoInput>` element by adding a [the ino-icon component and providing the slot `icon-trailing`](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--icons). Play around with each component's playground to see all possibilities.
```

```jsx
<InoInput label="password" outline>
  <InoIcon slot="icon-trailing" icon="password"></InoIcon>
</InoInput>
```

#### CSS Custom Properties

> For css customization, we can use **CSS Custom Properties** given by our `<InoInput>` component to style our outlines and labels.

Reviewer: ```suggestion
In order to provide an CSS way of styling the inovex-elements, we sometimes provide custom properties a.k.a. CSS-Variables.
In the case of the `<ino-input>`, there are two variables we can use the change the appearance of our component: `--ino-input-line-color` to change the color of the underline and the `--ino-input-label-color` which changes the color of the floating label. They can be used like this:
```

        timeout={5000}
        type="error"
        stay-visible-on-hover="false"
      >
        Login failed
      </InoSnackbar>
    );
  } else {
    return (
      <InoSnackbar
        action-text="Welcome back!"
        timeout={5000}
        type="success"
        stay-visible-on-hover="false"
      >
        Sucessfully logged in
      </InoSnackbar>
    );
  }
};
```

```jsx
{
  showSnackbar ? <ValidateLogin /> : null;
}
```

Change the types accordingly to your wishes. The timeout property is in _ms_ and indicates when the snackbar dissapears. The timeout can be disabled by setting it to a negative value.
More about the properties of the InoSnackbar is documented [here](https://elements.inovex.de/version/v7.1.0/?path=/docs/notification-ino-snackbar--playground).

## Finishing touches

### Add a logo

> Hint: automatically style your logo with rounded borders by using the `rounded` property of `<InoImg>`

```jsx
<InoImg
  alt="Elements logo"
  decoding="async"
  src={logo}
  height={75}
  width={75}
  fallbackIcon="cloud"
></InoImg>
```

If our image can't be loaded, it will render an `<InoIcon icon="cloud">` due to the `fallbackIcon="cloud"` property.

> The `decoding` property can either be `async`, `auto` or `sync` and sets the decode method of the native html input element.

Reviewer: ```suggestion
```

I think we can leave this part out as it might be too complicated for a beginner.
